h2o-core/src/main/java/water/fvec/RollupStats.java

```diff
@@ -217,6 +208,24 @@ else if (c instanceof C8DChunk)
     return this;
   }
 
+  private long processUUIDchunk(Chunk c, long checksum, long start, long l) {
+    for( int i=c.nextNZ(-1); i< c._len; i=c.nextNZ(i) ) {
+      if( c.isNA(i) ) _naCnt++;
+      else {
+        try {
+          long lo = c.at16l(i), hi = c.at16h(i);
+          if (lo != 0 || hi != 0) _nzCnt++;
+          l = lo ^ 37*hi;
+        } catch (Chunk.WrongType wt) {
```

Comment:
This is in parallel with checking the cause of the original failure: when chunks are created, vec type is just being ignored. So this part is to make sure we don't fail on bad data (it's just stats after all), but more checks are needed in NewChunk. What I wrote is a prototype.